dnsserv_test.go

```diff
@@ -1,40 +0,0 @@
-package main
-
-import (
-	"bytes"
-	. "gopkg.in/check.v1"
-	"net"
-	"testing"
-)
-
-func Test(t *testing.T) { TestingT(t) }
-
-type DNSNameserverSuite struct{}
-
-var _ = Suite(&DNSNameserverSuite{})
-
-func (s *DNSNameserverSuite) TestUDPConnection(c *C) {
```

Comment:
Since we have changed the port 3 times in the last 3 days and it can be different for each of us depending on what we are running on 8888 or some other number. But yeah, we can implement a test for this.

Instead of focusing on creating unit tests around this functionality, we will get more value from sending messages to real nameservers and seeing their responses. After we can do that, we will be able to make unit tests. Right now, we are not providing value by testing that a function can parse the bytes from "TEST CONNECTION" because that functionality is already provided by GoLang.

My point is that we don't know enough about what the bytes look like to create a rigid unit test for them.